.github/workflows/bump_packages.yml

```diff
@@ -2,6 +2,11 @@ name: Update packages
 
 on:
   workflow_dispatch:
+    inputs:
+      package:
+        description: 'Limit bumping to this package'
```

Comment:
Some bikeshedding: should this suggest it's a substring match?

Reply:
like this?